PySide2-stubs/QtWidgets.pyi

```diff
@@ -673,7 +673,7 @@ class QAbstractSpinBox(PySide2.QtWidgets.QWidget):
     def resizeEvent(self, event:PySide2.QtGui.QResizeEvent) -> None: ...
     def selectAll(self) -> None: ...
     def setAccelerated(self, on:bool) -> None: ...
-    def setAlignment(self, flag:PySide2.QtCore.Qt.Alignment) -> None: ...
+    def setAlignment(self, flag:PySide2.QtCore.Qt.Alignment|PySide2.QtCore.QtAlignmentFlag) -> None: ...
     def setButtonSymbols(self, bs:PySide2.QtWidgets.QAbstractSpinBox.ButtonSymbols) -> None: ...
```

Comment:
PySide2.QtCore.QtAlignmentFlag sounds like a typo. It should be PySide2.QtCore.Qt.AlignementFlag (notice the extra dot at the end) no ?

I am surprised how it works at all given all this.

Reply:
I don't get why I don't get an error running Mypy. New pull request on the way.